licenses/library_test.go

```diff
@@ -69,6 +71,15 @@ func TestLibraries(t *testing.T) {
 				"github.com/google/go-licenses/licenses/testdata/indirect",
 			},
 		},
+		{
+			desc:       "Binary dependency",
```

Comment:
Thanks for adding this test, it makes it much clearer why you're making this change :)